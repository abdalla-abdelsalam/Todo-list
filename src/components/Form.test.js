import {shallow} from 'enzyme'
import React from 'react'
import Form from './Form'
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

Enzyme.configure({ adapter: new Adapter() });

it('expect to render Form component',()=>{
    expect(shallow(<Form/>)).toMatchSnapshot();
})