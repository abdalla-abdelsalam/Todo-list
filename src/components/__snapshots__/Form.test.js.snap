// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Form component 1`] = `ShallowWrapper {}`;
